pkg/backup/backup/backup_tracker.go

```diff
@@ -95,14 +136,28 @@ func (bt *backupTracker) removeLogBackup(ns, name string) {
 
 // getLogBackupTC gets log backup's tidb cluster info.
 func (bt *backupTracker) getLogBackupTC(backup *v1alpha1.Backup) (*v1alpha1.TidbCluster, error) {
-	ns := backup.Namespace
-	name := backup.Name
-	clusterNamespace := backup.Spec.BR.ClusterNamespace
+	var (
+		ns               = backup.Namespace
+		name             = backup.Name
+		clusterNamespace = backup.Spec.BR.ClusterNamespace
+		tc               *v1alpha1.TidbCluster
+		err              error
+	)
 	if backup.Spec.BR.ClusterNamespace == "" {
 		clusterNamespace = ns
 	}
-	tc, err := bt.deps.TiDBClusterLister.TidbClusters(clusterNamespace).Get(backup.Spec.BR.Cluster)
+
+	err = retry.OnError(retry.DefaultRetry, func(e error) bool { return e != nil }, func() error {
+		tc, err = bt.deps.Clientset.PingcapV1alpha1().TidbClusters(clusterNamespace).Get(context.TODO(), backup.Spec.BR.Cluster, metav1.GetOptions{})
+		if err != nil {
+			klog.Warningf("get log backup %s/%s tidbcluster %s/%s failed and will retry, err is %v", ns, name, clusterNamespace, backup.Spec.BR.Cluster, err)
+			return err
+		}
+		return nil
+	})
+
 	if err != nil {
+		klog.Errorf("get log backup %s/%s tidbcluster %s/%s failed and will retry, err is %v", ns, name, clusterNamespace, backup.Spec.BR.Cluster, err)
 		return nil, fmt.Errorf(fmt.Sprintf("get log backup %s/%s tidbcluster %s/%s failed", ns, name, clusterNamespace, backup.Spec.BR.Cluster), err)
```

Comment:
It seems the returned error has problems. You don't need use `fmt.Sprintf` in `fmt.Errorf`, and there is no placeholder for the `err` in the result of `fmt.Sprintf`.

```suggestion
		return nil, fmt.Errorf("get log backup %s/%s tidbcluster %s/%s failed, err %s", ns, name, clusterNamespace, backup.Spec.BR.Cluster, err)
```

Reply:
ok